src/ShopBundle/objects/db/TShopPaymentHandler/TShopPaymentHandlerPayPal_PayViaLink.class.php

```diff
@@ -219,7 +245,12 @@ public function ProcessIPNRequest($oOrder, $aURLParameter)
                 $bPaymentOk = true;
                 $bPaymentCompleted = true;
             } else {
-                $logger->error("PayPal IPN: invalid amount paid: required = {$oOrder->fieldValueTotal}, paid = {$dPaymentValue}. Rawdata: ".print_r($aURLParameter, true));
+                $logger->error(
+                    "PayPal IPN: invalid amount paid: required = {$oOrder->fieldValueTotal}, paid = {$dPaymentValue}.",
+                    [
+                        'url_parameters' => $aURLParameter
```

Comment:
Please move the variable parts of the message to the context and add the order id and order number to the context as well

Reply:
ok